docs/validator/economics.md

```diff
@@ -0,0 +1,134 @@
+---
+id: economics
+title: The Economics of a Validator
+sidebar_label: Understanding Economics
+---
+## Overview
+
+NEAR is a sharded, developer-friendly, proof-of-stake blockchain. It will use a technique called 'dynamic sharding' that splits the network, so that much of the computation is done in parallel. With the increase of users and their transactions, NEAR will dynamically introduce additional shards.
+Therefore, the total number of validators and their assigned shards will change over time, based on the network usage (see [Nightshade documentation pages](../technical/nightshade.md)).
+However, each shard allows a fixed number of *seats*, with validators randomly assigned to one (or more) of them between every `epoch`. The more the shards, the higher the number of available *seats* - so, the more the *seats*, the larger the room for new validators.
+
+As a validator, you need to secure three classes of resources:
+1. NEAR tokens to stake
+2. Nodes infrastructure
+3. Continuous operation
+
+
+## 1. NEAR tokens to stake
+
+NEAR protocol determines validators at the end of each `epoch`, ordering them by the size of their stake. Current validators are re-elected automatically, by *re-staking* their tokens plus the accrued rewards; new validators may replace an existing one if they offer a more significant stake.
+
+As a validator, you can obtain a *seat* by:
+- Staking your NEAR tokens
+- Staking delegated NEAR tokens
+
+<blockquote class="info">
+    <strong>did you know?</strong><br><br>
+    A large enough stake can obtain more than one seat, multiplying the rewards. 
+</blockquote>
+
+### Stake your NEAR tokens
+
+Use [NEAR Shell](../development/near-clitool.md) to issue a staking transaction and append your transaction to the `proposals`. At the end of the `epoch`, if your proposal is sufficient, your node will become a validator - and will begin to generate chunks/blocks after one more epoch. You will be able to monitor the `proposals`, and eventually top-up your transaction to increase your odds.
+
+### Stake delegated NEAR tokens
+
+NEAR leverages *smart contracts* for delegation. Therefore, every validator should implement its staking contract, or use one already available among the community. 
```

Comment:
Not every validator needs to do this. Only those who wish to offer delegation or delegate their stake.